Doc/library/typing.rst

```diff
@@ -878,6 +878,34 @@ The module defines the following classes, functions and decorators:
       The ``_field_types`` and ``__annotations__`` attributes are
       now regular dictionaries instead of instances of ``OrderedDict``.
 
+.. class:: TypedDict(dict)
+
+   A simple typed name space. At runtime it is equivalent to
```

Comment:
"Namespace" is usually written as one word (Google says "Did you mean: namespace?").